sdk/mediaservices/Azure.Media.Analytics.Edge/api/Azure.Media.Analytics.Edge.netstandard2.0.cs

```diff
@@ -259,21 +273,53 @@ public partial class MediaGraphMotionDetectionProcessor : Azure.Media.Analytics.
     }
     public partial class MediaGraphNodeInput
     {
-        public MediaGraphNodeInput() { }
+        public MediaGraphNodeInput(string nodeName) { }
         public string NodeName { get { throw null; } set { } }
```

Comment:
If you are taking the `NodeName` as a ctor parameter the property shouldn't be settable.

Reply:
This is autogenerated code. Not sure that we can control this unless we change the actual autorest file itself?

Reply:
if you copy the property into your customization file the generator wouldn't generate it anymore and you can customize it at will.

Reply:
This is happening in many places though. For example look at the property instance in MediaGraphInstanceSetRequest.cs.

Reply:
Ah, my bad. I was under the impression you've added a customized ctor. Ignore me.